src/main/java/ch/uzh/ifi/hase/soprafs21/repository/UserRepository.java

```diff
@@ -23,4 +23,7 @@
 
     @Query("select u from User u where within(u.lastUserLocation, ?1) = true")
     List<User> findByArea(Polygon areaFilterPolygon);
+
+    @Query("select u from User u where u.id <> ?1")
+    List<User> findAll(Long userId);
```

Comment:
Rename this arg. to something like `userIdToExclude`, else it's a bit confusing.

Reply:
Agree